cypress/productAnalytics/index.ts

```diff
@@ -173,7 +173,7 @@ export const dashboard = {
         cy.get('.PropertyFilterButton').should('have.length', 0)
         cy.get('[data-attr="property-filter-0"]').click()
         cy.get('[data-attr="taxonomic-filter-searchfield"]').click().type('Browser').wait(1000)
-        cy.get('[data-attr="prop-filter-event_properties-0"]').click({ force: true })
+        cy.get('[data-attr="prop-filter-event_properties-0"]').click({ force: true }).wait(1000)
```

Comment:
Test was flapping because the previous input hadn't been torn down quick enough. Unclear why this change slowed things down but 🤷